server/app/routes/api/v1/va/serializers.js

```diff
@@ -8,139 +8,140 @@ import PhoneNumber from 'awesome-phonenumber'
  */
 
 function serializeAddress(address) {
-  if (!address) return ''
-  let keys = ['address1', 'city', 'state', 'zip']
+  if (!address) return ""
+  let keys = ["address1", "city", "state", "zip"]
```

Comment:
Could you do `prettier` runs as separate formatting-only PRs in the future so it's easier to review just your intentional changes?

Reply:
added - https://github.com/colab-coop/HelloVoter/pull/141